libminifi/test/integration/IntegrationBase.h

```diff
@@ -61,6 +61,10 @@ class IntegrationBase {
     return configuration;
   }
 
+  void setConfiguration(std::shared_ptr<minifi::Configure> configuration) {
+    this->configuration = configuration;
```

Comment:
I'd rather `std::move` the parameter if it's stored.

Reply:
Updated in aebe4fbd063ef08ba9877d0fb1657b44486a89a6